lib/docs/token-return-types.md

```diff
@@ -0,0 +1,71 @@
+# Token Return Types
+
+When acquiring tokens, MSAL JS v1 considers three different Token Request types:
+
+| Token Type | Definition |
+| ------- | -------- |
+| **token** | Indicates a request will be made for an Access Token |
+| **id_token** | Indicates a request will be made for an ID Token |
+| **id_token token** | Indicates that both an ID Token and Access Token will be requested |
+
+The following table describes how MSAL JS v1 determines which token request type is set and therefore what kind of token is returned by the API in question. 
+
+| Scenarios | ClientId as Only Scope | No Scopes | Access Token Scopes with No ClientId | Access Token Scopes with ClientId
+| ------- | ------- | -------- | --------- | -------- |
+| **loginRedirect/Popup** | id_token | id_token |id_token token | id_token token
+| **acquireTokenInteractive** | id_token | Error | token | id_token token 
+| **acquireTokenSilent** | id_token | Error | token | id_token token
+| **loginRedirect/Popup with accounts not matching** | id_token | id_token | id_token token | id_token token
+| **acquireTokenInteractive with accounts not matching** | id_token | Error | id_token token | id_token token
+| **acquireTokenSilent with accounts not matching** | id_token | Error | id_token token | id_token token
+
+As described in the table above, the Token Request Type is determined based on specific combinations of the following attributes:
+
++ `Scenario`: Each row is a combination of the public API that is called in the client application and whether or not the account passed in matches the account in the MSAL cache.
```

Comment:
The `account` is not always passed. Since we do not support multiple accounts in `msal@1.x.x`, we do look up the current `account` in the cache, when nothing is passed. Please verify this and update the statement accordingly.